src/App.tsx

```diff
@@ -1,204 +1,138 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { ErrorNotification } from './components/ErrorNotification';
+import { SearchField } from './components/SearchField';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  // #region searchField states
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  // const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [todosLoadingError, setTodosLoadingError] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [addError, setAddError] = useState(false);
+  const [deleteError, setDeleteError] = useState(false);
+  const [updateError, setUpdateError] = useState(false);
+  // #endregion
+
+  // #region useEffect
+  useEffect(() => {
+    console.log('first render');
```

Comment:
Remove it 
<img width="586" alt="image" src="https://github.com/user-attachments/assets/53908b10-ded3-48a7-95d5-a77ceba557c7" />